analysis/runtime/src/events.rs

```diff
@@ -70,6 +76,11 @@ pub enum EventKind {
 
     Offset(Pointer, isize, Pointer),
 
+    /// Marks the start of events in a new function body.
+    /// Used to distinguish address-taken locals that treated
+    /// as copies and ones that aren't.
+    BeginBody,
```

Comment:
Would `BeginFunc` make more sense?  `Body` is not as clear without `Func`, and with `Func`, `Body` is kind of redundant.

Reply:
MIR refers to it as a Body, which is why it's not Func here.

Reply:
Can we name it `BeginFuncBody` then?